Cargo.lock

```diff
@@ -442,13 +442,13 @@ dependencies = [
  "parity-scale-codec",
  "scale-info",
  "serde",
- "sp-arithmetic 16.0.0",
- "sp-core 21.0.0",
- "sp-io 23.0.0",
+ "sp-arithmetic 7.0.0",
+ "sp-core 8.0.0",
+ "sp-io 8.0.0",
  "sp-runtime 24.0.0",
  "sp-runtime-interface 17.0.0",
- "sp-std 8.0.0",
- "sp-trie 22.0.0",
```

Comment:
I'm not sure what went wrong, but doing 
`cargo update -p avail-core -p avail-subxt -p kate-recovery` 
reverts versions to correct ones (at least on my machine).

Reply:
Thank you!